saleor/graphql/payment/types.py

```diff
@@ -152,3 +152,14 @@ def resolve_credit_card(root: models.Payment, _info):
         if not any(data.values()):
             return None
         return CreditCard(**data)
+
+
+class PaymentInitialized(graphene.ObjectType):
+    class Meta:
+        description = "Represents a initialized data by payment gateway."
+
+    gateway = graphene.String(description="Payment gateway.", required=True)
+    name = graphene.String(description="Payment gateway name.", required=True)
+    data = graphene.JSONString(
+        description="Initiliazed data by gateway.", required=False
```

Comment:
Can we add a word or two about what this data is needed for? From the descriptions in this mutation I have no idea what this mutation is needed for.